cpp/src/bitmask/bitmask_ops.cu

```diff
@@ -13,8 +14,9 @@
 #include <thrust/device_vector.h>
 
 
+
 gdf_error all_bitmask_on(gdf_valid_type * valid_out, gdf_size_type & out_null_count, gdf_size_type num_values, cudaStream_t stream){
-	gdf_size_type num_chars_bitmask = gdf_get_num_chars_bitmask( num_values );
+	gdf_size_type num_chars_bitmask = gdf_num_bitmask_elements( num_values );
```

Comment:
rename num_chars_bitmask to num_bitmask_elements or something that does not refer to chars